src/app/core/components/spinner/spinner.component.html

```diff
@@ -0,0 +1,3 @@
+<div *ngIf="loadingService.loading$ | async" class="spinner-container">
```

Comment:
please, do not use services directly in template...

Reply:
Got it, thanks.